src/util/soundfile.cpp

```diff
@@ -101,7 +101,10 @@ namespace otto::util {
    * SoundFile Implementation
    */
 
-  SoundFile::SoundFile() {}
+  SoundFile::SoundFile() {
+    info.type = Info::Type::WAVE;
```

Comment:
nitpick: this should be a member initialization list

Reply:
Done.